ex35.py

```diff
@@ -63,49 +65,49 @@ def cthulu_room():
         cthulu_room()
 
 
-tries = 0 # how to get this into the function and used in the nested functions?

```

Comment:
Any time you're going to do something a limited number of times, or until some condition changes, use a loop.